pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -680,6 +680,26 @@
             + " non-backlog consumers as well.")
     private boolean dispatchThrottlingOnNonBacklogConsumerEnabled = false;
 
+    @FieldContext(
+            category = CATEGORY_POLICIES,
+            doc = "Default policy for publishing usage reports to system topic is disabled."
+            + "This enables publishing of usage reports"
+    )
+    private boolean resourceUsagePublishToTopic = false;
```

Comment:
Is it better to expose namespaceResourceLimiterEnabled or tenantResourceLimiterEnabled to users? the `resourceUsagePublishToTopic` is an internal mechanism, expose it to users is need the user to understand the details of his work

Reply:
When PIP-82 was presented in the community meeting, it was suggested that the mechanism used to exchange the usage information across brokers should be configurable/pluggable (for instance, one could use a third-party DB or zookeeper or something else). This config option was added for that purpose. There will be other configuration that allows the user to configure resource-groups to manage the quotas. That implementation is still work in progress and a PR will be raised when it is ready.

Reply:
@codelipenghui, I renamed the configuration option per your suggestion. Please take a look.

Reply:
@codelipenghui, please let me know if you have any more comments. If all looks good, can you please approve, so we can get the changes merged? Thank you.